addons/mail/models/mail_tracking_value.py

```diff
@@ -63,11 +63,14 @@ def _create_tracking_values(self, initial_value, new_value, col_name, col_info,
 
         :return: a dict values valid for 'mail.tracking.value' creation;
         """
-        tracked = True
-
         field = self.env['ir.model.fields']._get(record._name, col_name)
         if not field:
-            return
+            raise ValueError(
```

Comment:
same here (change to `UserError` or do not translate the error)

Reply:
Will keep an untranslated ValueError then